apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -289,7 +291,7 @@ private async Task<int> ExecuteMutationsAsync(CancellationToken cancellationToke
                 var mutations = GetMutations();
                 var transaction = Transaction ?? Connection.AmbientTransaction ?? new EphemeralTransaction(Connection, s_readWriteOptions);
```

Comment:
I've applied this in the latest version. This also has one additional benefit: It makes it very natural to include the `Priority` on the commit operation for other update statements in ephemeral transactions as well. The specs don't mention whether this should be done or not, but that is (probably) because ephemeral transactions are only present in .NET and JDBC.